align/view_test.go

```diff
@@ -19,6 +19,7 @@ var alignTests = []struct {
 	{"ACT", "ACGT", "AC-T\nACGT\n"},
 	{"CGCGCGCGCG", "CGCGCGTTTTCGCG", "CGCGCG----CGCG\nCGCGCGTTTTCGCG\n"},
 	{"CGCGCGCGCG", "CGAAAACGCGTTTTCGCG", "CG----CGCG----CGCG\nCGAAAACGCGTTTTCGCG\n"},
+	{"AA", "GGGAATT", "---AA--\nGGGAATT\n"},
```

Comment:
This relates to my comment above, but I think we might want a test that has both insertions and deletions in the final alignment.

Reply:
added tests